src/test/java/org/fundacionjala/coding/Christian/Kata4Test.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.christian;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "Kata4Test.java" should be located in "org/fundacionjala/coding/christian" directory, not in "/home/travis/build/AT-06/coding/src/test/java/org/fundacionjala/coding/Christian". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)